spec.md

```diff
@@ -91,13 +91,37 @@ perform a scheduled reboot.
 #### Event
 
 An "event" is a data record expressing an occurrence and its context. Events are
-routed from an event producer (the source) to interested event consumers. The
-routing can be performed based on information contained in the event, but an
-event will not identify a specific routing destination. Events will contain two
-types of information: the [Data](#data) representing the Occurrence and
+routed from an event producer to interested event consumers. The routing can be
+performed based on information contained in the event, but an event will not
+identify a specific routing destination. Events will contain two types of
+information: the [Data](#data) representing the Occurrence and
 [Context](#context) metadata providing contextual information about the
 Occurrence.
 
+#### Producer
+
+The "producer" is a specific instance, process or device that creates the data
+structure describing the event and its context.
```

Comment:
the term "data structure" sounds like it's implying the CloudEvent - if so, we should just say that. If not, can you elaborate on what you mean?